recipes/wcurl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "wcurl" %}
+{% set version = "2024.07.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/curl/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 962bb72e36e6f6cedbd21c8ca3af50e7dadd587a49d2482ab3226e76cf6dcc97
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - curl >=7.83.0
+
+test:
+  commands:
+    - wcurl --help
+    - wcurl https://raw.githubusercontent.com/conda-forge/curl-feedstock/master/LICENSE.txt
```

Comment:
Would it make sense to test for the file locally?

Reply:
Is there a recommend method to check for a file? The `file` command is not available in the Linux machine, so I'm wondering if conda-forge has any recommendation

Reply:
Please see [the simple existence test example]( https://conda-forge.org/docs/maintainer/adding_pkgs/#simple-existence-tests )

Reply:
This worked. Thank you!